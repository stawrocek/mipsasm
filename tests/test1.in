#this test tests different register naming conventions in mips
#and parser corner cases

.text 
main: 
ori $v0, $zero, 4
ori $v0, $0, 0x4

ori $v0,$zero,4
ori $v0,  ,$0,4  #test mars supported commas

ori $v0,$0,4#comment test

