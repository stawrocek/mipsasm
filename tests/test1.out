.text
main:
00400000	34020004		ori $v0, $zero, 4
00400004	34020004		ori $v0, $0, 0x4
00400008	34020004		ori $v0,$zero,4
0040000c	34020004		ori $v0,  ,$0,4
00400010	34020004		ori $v0,$0,


